interfaces/builtin/kvm.go

```diff
@@ -38,8 +48,60 @@ const kvmConnectedPlugAppArmor = `
 
 var kvmConnectedPlugUDev = []string{`KERNEL=="kvm"`}
 
+type kvmInterface struct {
+	commonInterface
+}
+
+var procCpuinfo = "/proc/cpuinfo"
+
+func getCpuFlags() (flags string, err error) {
+	buf, err := ioutil.ReadFile(procCpuinfo)
+	if err != nil {
+		// if we can't read cpuinfo, we want to know _why_
+		return "", fmt.Errorf("error: %v", err)
```

Comment:
Please provide a bit more context to the user here, something like: `cannot read cpu info: %v"` for the error string.